modules/workspace.nix

```diff
@@ -99,14 +108,27 @@ in
         Allows you to set wallpaper slideshow. Needs a directory of your wallpapers and an interval length.
       '';
     };
+
+    wallpaperPictureOfTheDay = lib.mkOption {
+      type = lib.types.nullOr wallpaperPictureOfTheDayType;
+      default = null;
+      example = "apod";
+      description = ''
+        Allows you to set wallpaper using the picture of the day plugin. Needs the provider and the fill mode.
```

Comment:
I don't see anything to do with the fill-mode in this pr. Is something missing?

Reply:
Oh sorry, actually I had added support for fill-mode but specifically just for potd, but I realised that we should be able to set it for all types of wallpapers. So I removed that bit (cuz that should be a different PR) but ig I forgot to update the description. Whoops.